backend/core/users/verification.py

```
@@ -0,0 +1,194 @@
+			socket_connect_timeout=5,
+			socket_timeout=5
+		)
+		# Validate connection immediately
+		client.ping()
+		return client
+	except Exception as exc:
+		# Fail fast on Redis misconfiguration in production
+		raise RuntimeError(
+			f"Failed to connect to Redis at {redis_url}. "
+			"Verify REDIS_URL is correct and Redis is running."
+		) from exc
```

Ruff (0.15.4)

[warning] Avoid specifying long messages outside the exception class

(TRY003)